### ISSUE TITLE
*log.safeError: stopper.go:182: runtime.plainError: assignment to entry in nil map

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#30513]
https://sentry.io/cockroach-labs/cockroachdb/issues/696906584/

```
*log.safeError: stopper.go:182: runtime.plainError: assignment to entry in nil map
  File "github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go", line 405, in runPostlude
  File "github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go", line 397, in runPrelude
  File "github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go", line 295, in RunTaskWithErr
  File "github.com/cockroachdb/cockroach/pkg/server/node.go", line 995, in batchInternal
  File "github.com/cockroachdb/cockroach/pkg/server/node.go", line 1036, in Batch
...
(16 additional frame(s) were not displayed)

stopper.go:182: runtime.plainError: assignment to entry in nil map
```

**Comment by jordanlewis** [comment]
Full stack trace:

```
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go in runPrelude at line 397
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go in RunTaskWithErr at line 295
github.com/cockroachdb/cockroach/pkg/server/node.go in batchInternal at line 995
github.com/cockroachdb/cockroach/pkg/server/node.go in Batch at line 1036
github.com/cockroachdb/cockroach/pkg/rpc/context.go in Batch at line 375
github.com/cockroachdb/cockroach/pkg/kv/transport.go in sendBatch at line 199
github.com/cockroachdb/cockroach/pkg/kv/transport.go in SendNext at line 1691
github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go in sendToReplicas at line 1303
github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go in sendRPC at line 387
github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go in sendSingleRange at line 454
github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go in sendPartialBatch at line 1080
github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go in func1 at line 1002
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go in func1 at line 385
```

**Comment by tschottdorf** [comment]
That's a weird one, it's in the stopper and indicates that the tasks map is empty. However, the stopper itself isn't nil or it would've crashed earlier.

https://github.com/cockroachdb/cockroach/blob/18519949df8e717c31a636bb153e526f45375ca8/pkg/util/stop/stopper.go#L397

We also definitely initialize this map in `NewStopper` and our code base shows no indication that we're creating a `&stop.Stopper{}` or `var x stop.Stopper` manually.

So, basically, what the heck? The node had "<1h" of uptime (so it had been running on the order of minutes at least).

**Comment by tschottdorf** [comment]
I got interrupted looking at this, but my latest thinking was that this was something silly about using a stopper after it's been destroyed.

**Comment by tbg** [comment]
Inactionable, sentry report expired.
